docs/en/administration/Query_planning.md

```diff
@@ -199,9 +199,9 @@ By removing the specific expressions (only keep the operators), the query plan c
 
 ## Query hint
 
-Query hints are directives or comments that explicitly suggest the query optimizer on how to execute a query. Currently, StarRocks supports two types of hints: variable-setting hint and join hint. Hints only take effect within a single query.
+Query hints are directives or comments that explicitly suggest the query optimizer on how to execute a query. Currently, StarRocks supports three types of hints: system variable hint, user-defined variable hint and join hint. Hints only take effect within a single query.
```

Comment:
system variable hint, user-defined variable hint and join hint -> system variable hints, user-defined variable hints, and join hints

Reply:
system variable (`SET_VAR`), user-defined variable hint (`SET_USER_VARIABLE`), and join hint.